### ISSUE TITLE
Idle default values

### ISSUE THREAD

**Issue by 125jdavis** [FOME-Tech/fome-fw#353]
![image](https://github.com/FOME-Tech/fome-fw/assets/24885463/a4e33f78-0ef2-444e-9c58-14f15890ee18)

Default closed loop min value was 0, so no negative idle correction was possible.  This may be an artifact of an old base tune, but should be reviewed.

**Comment by OrchardPerformance** [comment]
Absolutely, I think a full pass of default values is needed after the ts tips. 
Will put it on my todo list.
